demo/server/src/app.ts

```diff
@@ -14,6 +14,16 @@ import appConfig from './util/config';
 
 const app: express.Express = express();
 const config = appConfig.getConfig();
+let corsConfig = {};
+if (!config.keycloakConfig) {
+  const dynamicOrigin = function(origin, callback) {
```

Comment:
This is indeed equivalent to `"*"`, which is the default, so I also don't understand why we need this custom function.

We actually need to set cors for all origins because the behavior without it is not allowing cross-domain requests at all.